Doc/library/sqlite3.rst

```diff
@@ -1591,6 +1574,22 @@ Cursor objects
       including :abbr:`CTE (Common Table Expression)` queries.
       It is only updated by the :meth:`execute` and :meth:`executemany` methods.
 
+   .. attribute:: row_factory
+
+      Control how a row fetched from this :class:`!Cursor` is represented.
+      Set to :class:`sqlite3.Row` or a custom :term:`callable`
+      that accept two arguments,
+      a :class:`Cursor` object and the raw row results as a :class:`tuple`,
+      and return a custom object representing an SQLite row.
+      If ``None``, a fetched row is represented as a :class:`tuple`.
+
+      Defaults to what :attr:`Connection.row_factory` was set to
+      upon :class:`!Cursor` creation.
```

Comment:
```suggestion
      when the :class:`!Cursor` was created.
```

IMO, this wording is more direct and straightforward and easier to parse, likely particularly so for non-native readers.